ios/Helium.xcodeproj/project.pbxproj

```diff
@@ -12,7 +12,6 @@
 		04B7B04F50704A1AB50E9ED2 /* Inter-MediumItalic.otf in Resources */ = {isa = PBXBuildFile; fileRef = 81166484EECE40299BEB9570 /* Inter-MediumItalic.otf */; };
 		09C5A179F5234DFEA8A1BFEB /* Inter-Black.otf in Resources */ = {isa = PBXBuildFile; fileRef = 1B612616272949C8AF98D15A /* Inter-Black.otf */; };
 		0B74DE2CA1694927BF8B64BC /* NotoSerif-Bold.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 1007DBE6C5684F4999FA631B /* NotoSerif-Bold.ttf */; };
-		0E9226941D8EB508973B0E6A /* libPods-Helium-HeliumTests.a in Frameworks */ = {isa = PBXBuildFile; fileRef = CA5A5E66FD6140EE4B025D78 /* libPods-Helium-HeliumTests.a */; };
```

Comment:
Do you know why so much changed here?

Reply:
Not exactly. I had to do a lot of fighting to get it to build on my m1, I think this is just a side effect.